caf-audit-service-container/README.md

```diff
@@ -20,7 +20,30 @@ The number of replica shards (copies) that each primary shard should have. e.g.
 
 ##### CAF\_AUDIT\_SERVICE\_API\_CONFIG\_PATH
 
-The path to the directory containing the config.properties which can be used as an alternative means of specifying the environment variables. e.g. .\config-props 
```

Comment:
Readme for audit-service-deploy is in a separate pull request. (https://github.com/CAFAudit/audit-service-deploy/pull/5)

I'll update this readme and push the changes